### PR TITLE
More nav-buttons have been added, and page colors have been revamped. Filter buttons are also refined.

### PR DESCRIPTION
**Pull request by ContonJon**

### CODE REVIEW

File: src/components/Nav/index.tsx [hunk 1]
```diff
@@ -15,24 +21,32 @@ export const Nav = () => {
       display: "flex",
       alignItems: "center"
     }}>
-      <div style={{ backgroundColor: "#000000", width: "100%", height: "100%", position: 'absolute', opacity: "0.5", zIndex: "0" }}/>
+      <div style={{ backgroundColor: "#3E54AC", width: "100%", height: "100%", position: 'absolute', opacity: "1", zIndex: "0" }}/>
```
Comment (chrisj770) [inline src/components/Nav/index.tsx:24]: This was an excellent change! Color choice for the navigation bar looks great. 😎😎😎😎

Comment (ContonJon) [reply src/components/Nav/index.tsx:24]: Thank you Chris. Very cool

**Review by chrisj770 — APPROVED** [review]

File: src/components/Nav/index.tsx (hunk 1 above)
Comment (chrisj770) [reply src/components/Nav/index.tsx:24]: @lilygostovic Please review these beautiful changes ASAP lol

Comment (lilygostovic) [reply src/components/Nav/index.tsx:24]: Is it not letting you merge if one of you approves it?

**Review by lilygostovic — APPROVED** [review]

File: src/components/Nav/index.tsx (hunk 1 above)
Comment (chrisj770) [reply src/components/Nav/index.tsx:24]: Nah it requires one reviewer with "write access" which only you have 😅

Comment (lilygostovic) [reply src/components/Nav/index.tsx:24]: Ok I'll try to change that ASAP sorry about that

**Pull request merged**